posts/2023/mfdataset.ipynb

```diff
@@ -0,0 +1,8065 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        combine='nested',
</pre><p>Some explanation on the difference between combine='nested' and 'by_coords' would be helpful here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/esds/pull/238/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>